cmd/airflow.go

```diff
@@ -751,6 +756,11 @@ func airflowSettingsExport(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	if compose {
+		fmt.Println("Exporting compose file to " + composeFile)
+		return containerHandler.ComposeExport(settingsFile, composeFile)
```

Comment:
```suggestion
		fmt.Println("Exporting compose file to " + exportComposeFile)
		return containerHandler.ComposeExport(settingsFile, exportComposeFile)
```

Based on my above comment